src/components/ItemTooltip/index.jsx

```diff
@@ -235,7 +235,7 @@ const ItemTooltip = ({ item, inflictor }) => (
       }
       return null;
     })}
-    {item.hint && <Hint>{item.hint}</Hint>}
+    {item.hint && item.hint.map((hint) => <Hint>{hint}</Hint>)}
```

Comment:
Are there any cases where the hint is not an array yet? To prevent a crash should we do like hint?.map

Reply:
Good catch, I'll add that